packages/lib/src/components/Card/components/CardInput/CardInput.tsx

```diff
@@ -216,6 +216,17 @@ class CardInput extends Component<CardInputProps, CardInputState> {
                             </LoadingWrapper>
                         ) : (
                             <LoadingWrapper status={sfpState.status}>
+                                {hasHolderName && positionHolderNameOnTop && (
+                                    <CardHolderName
+                                        required={this.props.holderNameRequired}
+                                        placeholder={this.props.placeholders.holderName}
+                                        value={this.state.data.holderName}
+                                        error={!!this.state.errors.holderName}
+                                        isValid={!!this.state.valid.holderName}
+                                        onChange={this.handleHolderName}
+                                    />
```

Comment:
Maybe we could extract this into a variable to a void this duplication.

Reply:
Done!